googletest/include/gtest/gtest.h

```diff
@@ -116,6 +116,10 @@ GTEST_DECLARE_string_(color);
 // the tests to run. If the filter is not given all tests are executed.
 GTEST_DECLARE_string_(filter);
 
+// This flag controls whether Google Test installs a signal handler that dumps
```

Comment:
Could you please explain how this change is relevant to clang warnings. 
Thank you

Reply:
The macro expansion in `gtest.cc`, line 243 defines a variable named `FLAGS_gtest_install_failure_signal_handler`. Clang warns because the variable is non-static, but has not been declared extern.

This change adds the corresponding declaration. I'm assuming that this omission was merely an oversight, because all other flags are already declared in `gtest.h`, and there's no reason why only this one wouldn't be.